cloud/readme.md

```diff
@@ -0,0 +1,63 @@
+# UDMI terraform configuration files
+
+The files in this folder enable the creation of Cloud infrastructure to support the UDMI device-to-cloud data pipeline.
+
+At the moment only Google Cloud Platform (GCP) is supported.
+
+## Pre-requisites
+
+There are some pre-requisites that need to be satisfied in order to use this terraform configuration to setup UDMI infrastructure in a GCP cloud project:
+
+1. A working installation of [terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli?in=terraform/gcp-get-started)
+2. An [existing project on GCP](https://cloud.google.com/resource-manager/docs/creating-managing-projects)
+3. A Standard type storage bucket setup with the name `udmi-terraform-state-bucket`
+4. [Create a service account](https://cloud.google.com/iam/docs/creating-managing-service-accounts) for terraform with Owner role
+5. [Generate a JSON key](https://cloud.google.com/iam/docs/creating-managing-service-account-keys) for the terraform service account, download it and store it in the [`udmi/cloud/auth`](./auth) folder with the name `credentials.json`
+
+TODO: create a gcloud based shell script to automatically does all activities listed above.
+
+## Getting started
+
+To get started, copy the `terraform.tfvars.template` file to `terraform.tfvars`
+
+```
+cp terraform.tfvars.template terraform.tfvars
+```
+
+and edit it according to the project settings.
```

Comment:
personally I find this a bit too disjointed in terms of individual steps, I'd propose bigger blocks that have multiple commands per block. Often when I'm following a guide like this I just like to cut-and-paste, and if there's more in one block (e.g. all the cp commands) then it can be one step.

Reply:
sure, done.